wbooks/src/app/screens/dashboard/components/Search/index.js

```diff
@@ -0,0 +1,39 @@
+import React, {Component} from 'react';
+import './style.css';
+
+class Search extends Component {
+  state = {updateFilters : this.props.updateFilters};
+
+  handleTextChange(e){
+    this.setState({titleFilter: e.target.value})
+  }
+
+  handleSelectChange(e){
+    this.setState({dropdownFilter: e.target.value});
+  }
+
+  handleSubmit(e){
+    e.preventDefault();
+   this.props.updateFilters(this.state.dropdownFilter, this.state.titleFilter); 
+  }
+
+  render(){
+    return (
+      <form onSubmit={this.handleSubmit.bind(this)} className="search">
+        <select className="filter-dropdown" defaultValue="" onChange={this.handleSelectChange.bind(this)}>
+          <option value="">Seleccionar filtro </option>
+          <option value="short story">Short story</option>
```

Comment:
declare all values as contants in a new file. `Search/constants.js` and use the constants here
`<option value={CONSTANT}>`


Example of constants.js
```
export const SUSPENSE = "suspense";
export const FICTION = "fiction";
...
```